src/arevtur/xElements/itemListing/ItemListing.js

```diff
@@ -27,6 +27,13 @@ customElements.define(name, class extends XElement {
 	}
 
 	connectedCallback() {
+		this.$('#copy-item-button').addEventListener('click', e => {
+			this.itemData_.valueBuild.then(valueBuild => {
+				valueBuild = valueBuild.substring(0, valueBuild.indexOf('Note:'));
```

Comment:
Copy the item so you can ctrl+V in PoB to import the item easily. I wasn't sure how to get rid of the extra text in the items I was seeing, substring cleaned it up.

Reply:
Ok, that ("Copy the item so you can ctrl+V in PoB to import the item easily") sounds good.
But using valueBuild and substring isn't ideal because:
- it's async
- it will include the PoB tooltip meaning some mods will be applied twice
- for items that don't include 'note', this will copy an empty string

Instead, i suggest:
```
e => {
  navigator.clipboard.writeText(this.itemData_.text);
  e.stopPropagation();
}
```

and in datafetcher.js near line 280
```
		let text = ItemEval.decode64(itemData.item.extended.text);
		let valueBuild = itemEval?.evalItem(text) || Promise.resolve('');
```
and near line 312 include `text` between `priceDetails` and `debug`.

Reply:
That worked really nicely